### PR TITLE
Rotating Box

### PR DESCRIPTION
**Pull request by nehalskdev**

### CODE REVIEW

**Review by Rehan-Khan-Engineer — APPROVED** [review]
Nice @nehalskdev , If possible consider responsiveness as well.

**Comment by nehalskdev** [comment]
> Nice @nehalskdev , If possible consider responsiveness as well.

please drop me some suggestions how i can make this one responsive

File: style.css [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+@keyframes roating {
+  0% {
+      background-color: rgb(12, 12, 11);
+      transform: scaleX(5%);
+  }
+  25% {
+      opacity: .2;
+      transform: scaleX(50%);
+  100% {
+
```
Comment (Rehan-Khan-Engineer) [inline style.css]: I think you forgot the closing bracket? @nehalskdev

**Comment by Rehan-Khan-Engineer** [comment]
> > Nice @nehalskdev , If possible consider responsiveness as well.
> 
> please drop me some suggestions how i can make this one responsive

Sure, Keep it simple, focus on height, width and font size properties. (maybe you can use vh and vw units!). And i would personally not prefer writing "NEHAL SHAIKH" directly  inside div tag , (maybe you can put it inside p or h5 etc.!)

Try, i think you can do it. 💪 @nehalskdev

File: style.css (hunk 1 above)
Comment (nehalskdev) [reply style.css]: it is fixed please check.

**Comment by nehalskdev** [comment]
> > > Nice @nehalskdev , If possible consider responsiveness as well.
> > 
> > 
> > please drop me some suggestions how i can make this one responsive
> 
> Sure, Keep it simple, focus on height, width and font size properties. (maybe you can use vh and vw units!). And i would personally not prefer writing "NEHAL SHAIKH" directly inside div tag , (maybe you can put it inside p or h5 etc.!)
> 
> Try, i think you can do it. 💪 @nehalskdev

i have made it responsive please do checkout

**Review by syedabdullahali — APPROVED** [review]

**Review by syedabdullahali — COMMENTED** [review]
nice one

**Review by mmusk — APPROVED** [review]
Instead of writing the names of the colours directly, use hex code it's a good practise also place that text in a `<p>` tag instead

**Comment by nehalskdev** [comment]
> Instead of writing the names of the colours directly, use hex code it's a good practise also place that text in a `<p>` tag instead 

Surely will keep that in mind from next time @mmusk

**Comment by OSAMA478** [comment]
nice....

**Review by OSAMA478 — APPROVED** [review]
well done...

**Review by syedabdullahali — APPROVED** [review]

**Review by syedabdullahali — APPROVED** [review]

**Pull request merged**